schemas/GO/GO_edges_consider.yaml

```diff
@@ -1,6 +1,7 @@
----
 name: GO_edges_consider
```

Comment:
Note there's only 1 GO edges collection at this point.

Out of scope, but the GO and ENVO schemas are not exactly aligned to what the delta loader is producing